### PR TITLE
Feature/logs api

### LINKED ISSUE #134: NODE - Logs API
__Logs API -> /logs__

Allowed Roles | Description | Method | Endpoint
-- | -- | -- | --
JF | Logs - Find All | GET | /
JF | Logs - Find All for Student | GET | /students/:student_id
JF | Logs - Find All for Students in Company | GET | /students/company/:company_id
JF | Logs - Find One | GET | /log_id

### Checklist
- [x] Find All
- [x] Find All for Student
- [x] Find All for Students in Company
- [x] Find One

### PR DESCRIPTION
**Pull request by sbenemerito**
Closes #134

### CODE REVIEW

File: backend_node/app.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ app.listen(PORT, () => {
     console.log(`Server listening on port ${PORT}`);
 });
 
+// TODO: Clean routing
```
Comment (hugh-n) [inline backend_node/app.js]: Don't forget to remove once finished.

File: backend_node/controllers/task.controller.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      res.json(log);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json({
+        success: false,
+        msg: "Something went wrong: " + err
+      });
+    });
+};
```
Comment (hugh-n) [inline backend_node/controllers/task.controller.js]: Controller is task but findAll is LoginLog? Or is this just temporary code?

Comment (hugh-n) [reply backend_node/controllers/task.controller.js]: If this is temporary code, its better to remove this for now from the commit if you can.

Comment (sbenemerito) [reply backend_node/controllers/task.controller.js]: My bad, this is a mistake. Really sorry, file has been removed.

**Pull request merged**